internals/cli/cmd_help.go

```diff
@@ -164,8 +169,8 @@ type HelpCategory struct {
 	Commands    []string
 }
 
-// helpCategories helps us by grouping commands
-var helpCategories = []HelpCategory{{
+// HelpCategories helps us by grouping commands
+var HelpCategories = []HelpCategory{{
```

Comment:
Is there a reason to expose the whole thing as opposed to only allowing add ?

Reply:
It is a common idiom in Go, and allows us to tinker with the categories in different ways (e.g. removing some commands from the help manual on certain applications); besides, that was the behavior that was agreed upon last meeting (see 2023/06/13 meeting notes).